src/ui/ui.js

```diff
@@ -116,8 +61,17 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             storyContent: ".sjrkc-st-story-content",
             storyTags: ".sjrkc-st-story-tags"
         },
+        // The name of the model change source when block order is updated.
+        // This should match the source specified in the block modelListeners
+        orderUpdateSource: "orderUpdate",
         events: {
-            onStoryUpdatedFromBlocks: null
+            onStoryUpdatedFromBlocks: null,
+            onBlockManagerCreated: null,
+            onReadyToBind: {
```

Comment:
Yes, the intent is to use it as an aggregate event. I also noticed that `onBlockManagerCreated` would always fire after `onTemplateRendered` for `ui` grades with a `blockManager` component (instances of `storyUi`), but the intent is to keep `onReadyToBind` from firing the moment the template is done rendering (which was previously the case in `ui`), as that is too early for the `storyEditor` grade.

I thought this solution was a wiser choice than overriding the listener in the `templateManager` component of `ui`, and it allows for a bit more flexibility in configuring the timing. It might also be premature optimization :) Would you like me to revert to the previous setup and find another way to delay `onReadyToBind` for the `storyEditor`?

Reply:
It don't seem like a good idea to share the same `onReadyToBind` between `ui` and `storyUI` if one is dependent on the other and they are fired in sequence. I would suggest to use different events, such as `onUIReady` for `ui` component and `onStoryUIReady` for `storyUI` component.

Reply:
Okay, I'll separate the usage of them. I may still leave the template render event name as `onTemplateRendered`, since that's more accurate, and limit `onReadyToBind` usage to `storyUi` grades

Reply:
Feel free to use names that make sense. I've never been good at coming up with names.